web/skins/classic/css/classic/views/event.css

```diff
@@ -106,32 +106,32 @@
 #monitorStatus #monitorState {
 }
 
-#dvrControls {
+dvrControls {
```

Comment:
This is not correct... the # tells it to select by id.. without the # it is not valid css.

Reply:
I got it in my head that class selector was a bare name.  Meant to . those.  Checking if everything works as a class.  This explains why I thought your branch was styling the buttons differently than main.  I'd already broken it before I checked.